server/healthcheck.js

```diff
@@ -30,6 +30,7 @@ export const setupHealthcheckController = app => {
       'PUBLIC_BOT_ADDRESS',
       'PUBLIC_WEB_APP_ADDRESS',
       'PUBLIC_BOT_API_URL',
+      'PUBLIC_APP_VERSION',
```

Comment:
does not need a readiness check because this env variable is set at build time and will not be set in k8s